docs/tutorials/get-started-with-ngo.md

```diff
@@ -0,0 +1,619 @@
+
+```csharp
+       public override void OnNetworkSpawn()
+        {
+            if (IsOwner)
+            {
+                Move();
+            }
+        }
+```
+
+:::note
+
+You can override this behavior on any `NetworkBehaviour`.
+
+:::
+
+Since the server and client can be the same machine and the Player's owner (aka Host), you want further to differentiate the two and have different Move behavior for each.
+
+If the current player is the server, the code determines a random position to spawn the Player. You can’t determine the spawn location if the current player is the client; you have to get it from the server.
+
+```csharp
+       public void Move()
+        {
+            if (NetworkManager.Singleton.IsServer)
+            {
+                var randomPosition = GetRandomPositionOnPlane();
+                transform.position = randomPosition;
+                Position.Value = randomPosition;
+            }
+            else
+            {
+                SubmitPositionRequestServerRpc();
+            }
+        }
+
+[ServerRpc]
+        void SubmitPositionRequestServerRpc(ServerRpcParams rpcParams = default)
+        {
+            Position.Value = GetRandomPositionOnPlane();
+        }
+
+        void Update()
+        {
+            transform.position = Position.Value;
+        }
+```
+
+#### Add the `HelloWorldPlayer.cs` script to the Player prefab
+
+This section guides you through adding the `HelloWorldPlayer.cs` script to the Player prefab.
+
+Select the Player prefab:
+
+1. From the **Project** tab, select **Assets** > **Prefabs**.
+2. Select **Player**.
+
+Add the `HelloWorldPlayer.cs` script to the Player prefab as a component:
+
+1. With the Player prefab selected, select **Add Component** from the Inspector tab.
+2. Select **Scripts** > **Hello World** > **Hello World Player**.
+
+#### About the `HelloWorldPlayer` script
+
+The `HelloWorldPlayer` class inherits from `NetworkBehaviour` instead of `MonoBehaviour`.
+
+```csharp
+public class HelloWorldPlayer : NetworkBehaviour
+```
+
+The `HelloWorldPlayer` class defines a `NetworkVariable` to represent the player's networked position.
+
+```csharp
+public NetworkVariable<Vector3> Position = new NetworkVariable<Vector3>();
+```
+
+The `HelloWorldPlayer` class overrides `OnNetworkSpawn`.
+
+```csharp
+       public override void OnNetworkSpawn()
+        {
+            if (IsOwner)
+            {
+                Move();
+            }
+        }
+```
+
+Any `MonoBehaviour` implementing `NetworkBehaviour` can override the Netcode method `OnNetworkSpawn()`. The `OnNetworkSpawn()` method fires in response to the `NetworkObject` spawning. The `HelloWorldPlayer` class overrides `OnNetworkSpawn` since clients and the server run different logic.
+
+:::note
+
+You can override this behavior on any `NetworkBehaviour`.
```

Comment:
This and the above line seem redundant with what is in lines 381-387

Reply:
Good catch. I'll keep the first instance of the information.